### DISCUSSION TITLE
تست سرعت دانلود آی پی ها

### DISCUSSION THREAD

**Discussion by artankohan** [MortezaBashsiz/CFScanner#355] [Q&A]
راهی هست که بشه سرعت دانلود آی پی های پیدا شده ارو تست کرد؟

**Comment by tempookian** [comment]
لطفا مشخص کنید از چه نسخه‌ای استفاده می‌کنید. توی همه نسخه‌ها سرعت دانلود آیپی‌ها رو می‌تونید ببینید دیگه

**Reply by artankohan** [reply]
نسخه ی ویندوز.اخه فقط پینگ رو مینویسه. تا جایی که من سرچ کردم فهمیدم پینگ با سرعت دانلود تفاوت داره. درست نی؟
